### ISSUE TITLE
Allow for Configuring Standard Browser Settings

### ISSUE THREAD

**Issue by mattvisa** [linuxmint/webapp-manager#308]
I love using the web app tool in Linux Mint. However, each web app I create, I have to go in and setup the features of Brave or Firefox per website URL. If there were a way to set the settings universally for how a web app should be setup (ie, I go through and turn off telemetry on both Brave and Firefox, which are turned on by default) for each web app, it would be a really nice feature. As it stands now, you have to do it over and over again for each web app.

**Comment by djcrossan** [comment]
Yeah, I second this request! If  I use Firefox as the browser, I have to go and disable all Firefox's anti-privacy settings every time. If I use Librewolf, I have to go in and enable the save password option and it doesn't always stick. I'm sure there must be some simple way of tweaking the source profile that Web Apps uses, but I can't find any info on how to do this.

**Comment by McMork** [comment]
Yeah, I second this request! If I use Firefox as the browser, I have to go and disable all Firefox's anti-privacy settings every time. If I use Librewolf, I have to go in and enable the save password option and it doesn't always stick. I'm sure there must be some simple way of tweaking the source profile that Web Apps uses, but I can't find any info on how to do this.
